types/protoc-plugin/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for protoc-plugin 0.0
+// Project: https://github.com/konsumer/node-protoc-plugin/
+// Definitions by: Jonny Reeves <https://github.com/jonnyreeves>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import * as stream from "stream";
+import {
+  CodeGeneratorRequest as pb_CodeGeneratorRequest,
+  CodeGeneratorResponse as pb_CodeGeneratorResponse
+} from "google-protobuf/google/protobuf/compiler/plugin_pb";
+import { FileDescriptorProto, SourceCodeInfo } from "google-protobuf/google/protobuf/descriptor_pb";
+import Location = SourceCodeInfo.Location;
+
+export type OutputFiles = pb_CodeGeneratorResponse.File.AsObject[];
+
+export type SimplePluginCallback = (filesToGenerate: ReadonlyArray<FileDescriptorProto.AsObject>) => (OutputFiles | Promise<OutputFiles>);
+
+export default function simplePlugin(cb: SimplePluginCallback): Promise<void>;
```

Comment:
It's not `export default`: https://github.com/konsumer/node-protoc-plugin/blob/master/src/index.js#L82

Reply:
Thanks for taking the time to review @plantain-00,

I'm sorry I'm not overly familiar with authoring tsd, could you please point me, or suggest how I should express this export statement?

Thank you.

Reply:
Hi @plantain-00 I've had a stab at updating this to match what's being exported; please let me know if this is the best way to solve this problem.  Thanks :)